src/journal/meson.build

```diff
@@ -0,0 +1,104 @@
+# -*- mode: meson -*-
+
+libsystemd_journal_internal_sources = files('''
+  audit-type.c
+  audit-type.h
+  catalog.c
+  catalog.h
+  compress.c
+  compress.h
+  journal-def.h
+  journal-file.c
+  journal-file.h
+  journal-send.c
+  journal-vacuum.c
+  journal-vacuum.h
+  journal-verify.c
+  journal-verify.h
+  lookup3.c
+  lookup3.h
+  mmap-cache.c
+  mmap-cache.h
+  sd-journal.c
+'''.split())
+
+############################################################
+
+audit_type_includes = [config_h,
+                       missing_h,
+                       'linux/audit.h']
+if libaudit.found()
```

Comment:
This breaks if libaudit was disabled manually. In that case `libaudit` is set to `[]` and this fails with
`Arrays do not have a method called 'found'.`